Cargo.lock

```diff
@@ -3235,6 +3238,7 @@ dependencies = [
  "log",
  "ndk-sys",
  "num_enum",
+ "raw-window-handle 0.5.2",
  "raw-window-handle 0.6.0",
```

Comment:
I have no opinion on this duplication -- just want to make sure you're aware of it

Reply:
it's weird that it snuck in here but yeah, it's fine. We already had it duplicated for other reasons. Thanks for checking the Cargo.lock though!